std/os/openbsd/index.zig

```diff
@@ -175,6 +175,14 @@ pub fn realpath(noalias filename: [*]const u8, noalias resolved_name: [*]u8) usi
     return @bitCast(usize, if (c.realpath(filename, resolved_name) == null) isize(c._errno().*) else 0);
 }
 
+pub fn arc4random_buf(buf: [*]u8, nbytes: usize) !void {
+    c.arc4random_buf(buf, nbytes);
+    // Not happy about this
+    return switch (nbytes) {
+        else => error.Unexpected,
+    };
```

Comment:
The function arc4random_buf(3) is always successful but getRandomBytes expects an Error Union, thus this nonsensical  switch. Is there a better way to express this ?

Reply:
Here's the best way to solve this:
 * make `arc4random_buf` not have a possible error (because that is correct)
 * make `getRandomBytes` have an explicit error set (for an example of this see `posixRead` in the same file)

it won't be possible for `getRandomBytes` to return an error on OpenBSD (which is great! good job OpenBSD), but it is still correct to have the error union return type, because the purpose of `getRandomBytes` is to abstract over operating systems, and therefore portable code has to be ready for all the errors in its error set.

Reply:
Thanks.